src/list.js

```diff
@@ -130,7 +130,7 @@ class TypeInferedList extends BaseImmutableList {
   }
 
   get(index, notSetValue) {
-    return this[$store] ? this[$store].get(index, notSetValue) :
+    return this[$store] ? this[$store].get(parseInt(index), notSetValue) :
```

Comment:
Isn't the point of this library to not automatically type convert? Should it error here?